internal/crypto/cert.go

```diff
@@ -276,6 +277,9 @@ func (ca *CA) MakeClientCertificate(certFile, keyFile string, subject string, ex
 		AuthorityKeyId: ca.Config.Certs[0].SubjectKeyId,
 		SubjectKeyId:   clientPublicKeyHash,
 	}
+	if subject != "" {
```

Comment:
This is just to allow testing the behavior of the server when there is no subject on the client cert (make sure that we can't be crashed because of that, etc..)